README.md

```diff
@@ -218,6 +218,30 @@ The original wiki links markup is elaborate, but here we only support simple syn
 * Internal links:  `[[pageTitle]]`  or `[[pageTitle|displayText]]`. For example `[[Apple]]` gives `<a href="./Apple" title="Apple">Apple</a>`.
 * External links: `[https://example.com]` or `[https://example.com display text]`
 
+## Extending the parser
+
+Following example illustrates extending the parser to support more magic words
```

Comment:
May be we should avoid "magic words" and use "parse plugins". Magic word comes from mediawiki contexts and we cannot expect a user of this library aware of its meaning.